### PR TITLE
Comment should start with a whitespace and Uppercase

### PR DESCRIPTION
**Pull request by dapengJacky**
Signed-off-by: jacky <peng.chen@daocloud.io>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #
Comment should start with a whitespace and Uppercase

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
I agree that comments should start with a while space but with uppercase only when it's the comment for `exported` variable/function/methods, etc.

**Comment by dapengJacky** [comment]
> I agree that comments should start with a while space but with uppercase only when it's the comment for exported variable/function/methods, etc.
okay,I have updated,thanks for your advice

File: cmd/controller-manager/app/options/options.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ type Options struct {
 	// It can be set to "0" to disable the metrics serving.
 	// Defaults to ":8080".
 	MetricsBindAddress string
-	// concurrentClusterSyncs is the number of cluster objects that are
+	// ConcurrentClusterSyncs is the number of cluster objects that are
```
Comment (RainbowMango) [inline cmd/controller-manager/app/options/options.go:97]: nice catch!

But for others, I'm sorry I didn't make it clear.
For comments not for exported variables/functions/methods, there is no rule for should comment start with uppercase or not, so I guess no needs to update them.

Comment (dapengJacky) [reply cmd/controller-manager/app/options/options.go:97]: I agree with u,I didn't update them

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**